### ISSUE TITLE
Graphic flickering

### ISSUE THREAD

**Issue by Klauserus** [mgba-emu/mgba#1935]
Hello

I have discovered the following errors:

1. At Harvest Moon: Friends of Mineral Town

![bug2](https://user-images.githubusercontent.com/26491758/98468752-2d367f00-21dc-11eb-814c-a8937ebf1e2e.gif)

[logfile.txt](https://github.com/mgba-emu/mgba/files/5506550/logfile.txt)

2. Zelda. Minish Cap

![bug](https://user-images.githubusercontent.com/26491758/98468763-42aba900-21dc-11eb-8f6f-3ccb1f981b75.gif)

My PC Specs:

Win7 64bit, 650 ti GPU, CPU X4 955 BE AMD
mgba Branch: heads/0.8.4 Revision: 545a76f876af47671c8089b1ee09bf1d4a55f47e

**Comment by exelotl** [comment]
Hi, I don't think this is a problem with the emulator. The first gif seems to be an intentional effort by the game developers. Many games use flickering to achieve a semi-transparent effect by taking advantage of ghosting on the GBA's screen.

The effect _doesn't work_ on modern displays which don't have ghosting artifacts. Fortunately mGBA has an [interframe blending](https://mgba.io/2020/01/21/mgba-0.8.0/#interframe-blending) option which simulates the ghosting artifacts.

Try enabling this setting and see if this improves things for you:

![image](https://user-images.githubusercontent.com/569607/99998031-f18fdd80-2db5-11eb-9513-cda56af9a757.png)

**Comment by endrift** [comment]
That's true of the former, but I'm not sure about the latter example.

**Comment by profi200** [comment]
The first looks intentional indeed. For the second one i must be blind but i can't see anything wrong.

**Comment by endrift** [comment]
It's the door frame on the house at the end. The first 2/3 of the GIF doesn't have it.

**Comment by profi200** [comment]
Yeah, i first wanted to point that out but then i remembered how the scrolling across the screen in this game is not exactly smooth but rather "flickery" or however one can describe it. It's very noticeable with bigger screens and sharp scaling. I guess the "grass" bit got me confused.

**Comment by Klauserus** [comment]
Hello

i have tryed interlacing mode. In Harvest Moon there is a better result. but from time to time it like to be flickering, very short. 
(cant post this in a gif, because the positiv interlacing effect does not effect on a recorded it.)
